frontend/src/main/scala/bloop/dap/DebugSession.scala

```diff
@@ -178,6 +178,9 @@ final class DebugSession(
         response.command = Command.LAUNCH.getName
         attachedPromise.success(())
         super.sendResponse(response)
+      case "attach" =>
```

Comment:
Please, add comment that it is a response to an actual "attach" request and not our synthetic one

Reply:
I attend some more comment to both `sendResponse` and also `dispatchRequest` function, could you please take a look if it's better?